test/viewDirectiveSpec.js

```diff
@@ -324,7 +324,7 @@ describe('uiView', function () {
   });
 
   it('should instantiate a controller with controllerAs', inject(function($state, $q) {
-    elem.append($compile('<div><ui-view>{{vm.someProperty}}</ui-view></div>')(scope));
+    elem.append($compile('<div><ui-view></ui-view></div>')(scope));
```

Comment:
This test broke but not because of what it tests. The template property was not originally in the test and after this change the template property overwrites the initial view.